runtime/src/main/java/org/corfudb/runtime/collections/CorfuStore.java

```diff
@@ -274,21 +274,40 @@ void subscribe(@Nonnull StreamListener streamListener, @Nonnull String namespace
      *                         if null, only future updates will be returned
      */
     public void subscribe(@Nonnull StreamListener streamListener, @Nonnull String namespace,
-                   @Nonnull String streamTag, @Nonnull List<String> tablesOfInterest,
-                   @Nullable Timestamp timestamp) {
-        runtime.getTableRegistry().getStreamManager()
+                          @Nonnull String streamTag, @Nonnull List<String> tablesOfInterest,
```

Comment:
Deprecate the `subscribe` method that uses the old stream manager too ?

https://github.com/CorfuDB/CorfuDB/pull/2826/files#diff-f44ea07bfcd37e9243f12f8e2e3666056f2a25d4943965e359e61a43d4e64135R257

Reply:
Done, thanks for finding out.